drivers/pinmux/stm32/pinmux_stm32h7.h

```diff
@@ -50,19 +54,41 @@
 	(STM32_PINMUX_ALT_FUNC_8 | STM32_PUSHPULL_PULLUP)
 
 /* Port B */
+#define STM32H7_PINMUX_FUNC_PB6_I2C1_SCL    \
+	(STM32_PINMUX_ALT_FUNC_4 | STM32_OPENDRAIN_PULLUP)
+#define STM32H7_PINMUX_FUNC_PB6_I2C4_SCL    \
+	(STM32_PINMUX_ALT_FUNC_6 | STM32_OPENDRAIN_PULLUP)
 #define STM32H7_PINMUX_FUNC_PB6_USART1_TX   \
 	(STM32_PINMUX_ALT_FUNC_7 | STM32_PUSHPULL_PULLUP)
 #define STM32H7_PINMUX_FUNC_PB6_LPUART1_TX    \
-	(STM32_PINMUX_ALT_FUNC_6 | STM32_OPENDRAIN_PULLUP)
+	(STM32_PINMUX_ALT_FUNC_8 | STM32_OPENDRAIN_PULLUP)
```

Comment:
This is an issue, not related to I2C, which could be fixed in another commit

Reply:
@FRASTM done in 3077a04